main/test/ca/uwaterloo/flix/language/feature/TestChannelSelect.flix

```diff
@@ -0,0 +1,42 @@
+@test
+def select_int_channel_first(): Bool =
```

Comment:
Merge with other test file or renamed to `TestChannels` if this is for general test cases.

All tests should start with `testX` and use CamelCase.